pkg/partition/partition.go

```diff
@@ -90,9 +90,10 @@ func (d *Disk) createPartitionTable() error {
 }
 
 // addPartition is used to add a partition to the partition table.
-// Currently only a single partition can be created. i.e The method can be called only once for a disk
+// Currently only a single partition can be created i.e, The method can be called only once for a disk.
+// **label** is str argument that designates the partition uniquely.
 // TODO: @akhilerm, add method to create partition with given size
-func (d *Disk) addPartition() error {
+func (d *Disk) addPartition(label string) error {
```

Comment:
Also I would suggest to use a hard coded string like ``. An argument is not required.
```suggestion
func (d *Disk) addPartition() error {
	label := "OpenEBS_NDM"
```